README.md

```diff
@@ -3,91 +3,64 @@
 
 ## Requirements
 
-- Python 3.x
-- Postgres >= 9
+- [Docker](https://www.docker.com/)
+
 
 ## Running the app locally
 
-Perform the following steps from your terminal.
+We use Docker for local development. To get started, run the following from your terminal:
 
 1. Clone this repository and `cd` into your local copy.
 
     ```bash
     git clone git@github.com:datamade/committee-oversight.git
     cd committee-oversight
     ```
-2. Create a virtual environment. (We recommend using [`virtualenvwrapper`](http://virtualenvwrapper.readthedocs.org/en/latest/install.html) for working in a virtualized development environment.)
-
-    ```bash
-    mkvirtualenv committee-oversight
-    ```
-3. Install the requirements.
-
-    ```bash
-    pip install -r requirements.txt
-    ```
 
-4. Copy the example local settings file to the correct location:
+2. Run the application:
 
     ```bash
-    cp committeeoversight/local_settings.example.py committeeoversight/local_settings.py
+    docker-compose up
     ```
 
-5. Start a `hearings` database with a scrape from the [hearings repo](https://github.com/datamade/hearings) or a data dump from a pal. If you're following the latter path and have an archival format `hearings.dump` file (see [documentation](https://www.postgresql.org/docs/10/app-pgrestore.html)) in the root of this project, restore it by running:
+3. The root of this repository has a file named `hearings.dump`, a sample set of hearings data for use in local development. With `docker-compose up` still running, open a new terminal tab and run:
 
     ```bash
-    pg_restore -C -j4 --no-owner hearings.dump | psql
+    docker exec committeeoversight-postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump
     ```
 
-6. Run migrations:
+    Note: To accommodate the restoration of this hearings dump, migrations are not run automatically with `docker-compose up`. If you need to run them manually, you can run `docker exec committeeoversight python manage.py migrate`.
 
-    ```bash
-    python manage.py migrate
-    ```
-
-7. Make a superuser for so that you can access the admin interface:
+4. If you don't already have the `lugarcenter` development password, create a new superuser by running:
 
     ```bash
-     python manage.py createsuperuser
+    docker exec committeeoversight python manage.py createsuperuser
```

Comment:
got this error when i ran this command:

```bash
(committee-oversight) call-me-hank:committee-oversight hannah$ docker exec committeeoversight python manage.py createsuperuser
Superuser creation skipped due to not running in a TTY. You can run `manage.py createsuperuser` in your project to create one manually.
```

since you set `tty: true` in your `docker-compose.yml` file, you should be able to get around this by using `docker-compose run` instead. 

(as an aside, [you can do this with `docker exec` with the `-t` flag](https://docs.docker.com/engine/reference/commandline/exec/), it just feels redundant to remember when you've already got the service defined in a way that abstracts it away!)